.changeset/red-planets-hammer.md

```diff
@@ -0,0 +1,7 @@
+---
+'@backstage/catalog-model': patch
+'@backstage/plugin-catalog-backend': patch
+'@backstage/plugin-catalog': patch
+---
+
+Implemented missing support for the dependsOn/dependencyOf relationships between Component and Resource catalog model objects. Added support for generating the relevant relationships to the BuiltinKindsEntityProcessor, and added simple support for fetching relationships between Components and Resources for rendering in the system diagram. All catalog-model changes backwards compatible.
```

Comment:
Think we can maybe fix the formatting here a little bit?

Maybe things like BuiltinKindsEntityProcessor could be `BuiltinKindsEntityProcessor` and some line breaks to make the change log easier to read :)

Reply:
Added the formatting as requested.